### PR TITLE
fix(select): update style of select - TWIG-37

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-select/select.html.twig [hunk 1]
```diff
@@ -60,16 +66,34 @@
 <div class="ecl-form-group ecl-form-group--select">
   {% if _label is not empty %}
     <label
-      class="ecl-form-label{{ invalid ? ' ecl-form-label--invalid' }}{{ hide_label ? ' ecl-form-label--hidden'}}"
-      for="{{ _id }}"
+      {{ _id ? 'for="' ~ _id ~  '"' }}
```
Comment (planctus) [inline src/ec/packages/ec-component-select/select.html.twig]: mind that you have two spaces here: `_id ~  '"'`

File: src/ec/packages/ec-component-select/select.html.twig [hunk 2]
```diff
@@ -60,16 +66,34 @@
...
+    <div class="ecl-feedback-message">{% block invalid_text _invalid_text %}</div>
+  {% endif %}
+
+  <div class="ecl-select__container
+    {{- disabled ? ' ecl-select__container--disabled' -}}
+    {{- ' ecl-select__container--' ~ _width }}"
+  >
     <select
-      id="{{ _id }}"
+      {{ _id ? 'id="' ~ _id ~  '"' }}
```
Comment (planctus) [inline src/ec/packages/ec-component-select/select.html.twig]: same as before .. ;)

File: src/ec/packages/ec-component-select/select.html.twig (hunk 1 above)
Comment (papegaill) [reply src/ec/packages/ec-component-select/select.html.twig]: Thx, done.

File: src/ec/packages/ec-component-select/select.html.twig (hunk 2 above)
Comment (papegaill) [reply src/ec/packages/ec-component-select/select.html.twig]: done

**Pull request merged**